com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/PickingSpaceTransforms.hlsl

```diff
@@ -1,3 +1,7 @@
+#if 0
+// This file overrides a large amount of macros to be compatible with the old
+// Unity C++ picking. It is no longer necessary with the new SRP based picking.
+
 #ifndef UNITY_PICKING_SPACE_TRANSFORMS_INCLUDED
```

Comment:
Might be best to get rid of this rather than just "if 0" the contents (I realize that might mean removing include directives from files that you aren't otherwise touching, so I'm fine with it either way)